kms/api-client/asymmetric.py

```diff
@@ -13,71 +13,100 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.rom googleapiclient import discovery
 
-import base64
 import hashlib
 
 from cryptography.exceptions import InvalidSignature
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.primitives.asymmetric import ec, padding, utils
 
+from google.cloud import kms_v1
+from google.cloud.kms_v1 import enums
+
+
+# [START kms_create_asymmetric_key]
+def create_asymmetric_key(project_id, location_id, key_ring_id, crypto_key_id):
+    """Creates an RSA encrypt/decrypt key pair within a specified KeyRing."""
+
+    # Creates an API client for the KMS API.
+    client = kms_v1.KeyManagementServiceClient()
+
+    # The resource name of the KeyRing associated with the CryptoKey.
+    parent = client.key_ring_path(project_id, location_id, key_ring_id)
+
+    # Create the CryptoKey object template
+    purpose = enums.CryptoKey.CryptoKeyPurpose.ASYMMETRIC_DECRYPT
+    algorithm = enums.CryptoKeyVersion.CryptoKeyVersionAlgorithm.\
+        RSA_DECRYPT_OAEP_2048_SHA256,
```

Comment:
Remove trailing comma. It turns algorithm into a one-element tuple which makes the create_crypto_key call fail.